apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/utils/tracking.js

```diff
@@ -0,0 +1,26 @@
+// Ensure Tracks Library
+window._tkq = window._tkq || [];
+
+export const trackView = vertical_id => {
```

Comment:
Can we make this more generic and move to a root folder? I can imagine us wanting to track more metrics in future.

We could have a util which prefills `recordEvent `. That would leave it requiring the completion of:

`a8c_test_full_site_editing_{{SOME_STAT_METRIC}}`

and

` { vertical_id },`

Not a blocker.

Reply:
> a8c_test_full_site_editing_{{SOME_STAT_METRIC}}

This is discouraged as it prevents you from searching event names in codebases. 

I'd rather keep the implementation very specific for our part of the plugin as I don't have enough info about the bigger picture and ways we might use it in future.

Reply:
Fair enough.